package.xml

```diff
@@ -30,9 +30,13 @@
   <buildtool_depend>cmake</buildtool_depend>
   <buildtool_depend condition="$ROS_PYTHON_VERSION == 2">python-setuptools</buildtool_depend>
   <buildtool_depend condition="$ROS_PYTHON_VERSION == 3">python3-setuptools</buildtool_depend>
+  <buildtool_depend condition="$ROS_PYTHON_VERSION == 3">python3-build</buildtool_depend>
+  <buildtool_depend condition="$ROS_PYTHON_VERSION == 3">python3-wheel</buildtool_depend>
 
   <buildtool_export_depend>cmake</buildtool_export_depend>
   <buildtool_export_depend condition="$ROS_PYTHON_VERSION == 3">python3-setuptools</buildtool_export_depend>
+  <buildtool_export_depend condition="$ROS_PYTHON_VERSION == 3">python3-build</buildtool_export_depend>
+  <buildtool_export_depend condition="$ROS_PYTHON_VERSION == 3">python3-wheel</buildtool_export_depend>
```

Comment:
Probably need a dep here for pip too; on NixOS at least neither the module nor executable is bundled with Python: https://search.nixos.org/packages?query=pip